Scripts/Python/xTakableClothing.py

```diff
@@ -359,7 +359,8 @@ def OnNotify(self,state,id,events):
                 PtDebugPrint("xTakableClothing: Guild set to:", guildSDLValues[base])
             if not self.IItemInCloset():
                 avatar.avatar.addWardrobeClothingItem(base,ptColor().white(),ptColor().white())
-                PtSendKIMessage(kKILocalChatStatusMsg,PtGetLocalizedString("KI.Messages.NewClothing", [self.IGetItem(base).description]))
+                if hasattr(self.IGetItem(base), "description"):
+                    PtSendKIMessage(kKILocalChatStatusMsg,PtGetLocalizedString("KI.Messages.NewClothing", [self.IGetItem(base).description]))
```

Comment:
IGetItem is running the slow ClothingItem constructor, so let's only call it once:

```suggestion
                item = self.IGetItem(base)
                if hasattr(item, "description"):
                    PtSendKIMessage(kKILocalChatStatusMsg, PtGetLocalizedString("KI.Messages.NewClothing", [item.description]))
```